recipe/meta.yaml

```diff
@@ -36,6 +38,7 @@ test:
     - cf
   commands:
     - pip check
+    - python -c "import unittest; from cf.test import test_pp; unittest.main(test_pp)"
```

Comment:
I'm OK running all the tests. We avoided that in the past due to CI limitation but since we moved to Azure that is no longe true and running all tests is probably better. (No need to add it in this PR though.)

Reply:
I agree on running as many tests as possible. But I found that quite a few of their tests fail when I tried it locally - so for the moment probably we keep to only this test. I'll raise a Q with the original package developers about this though.